apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -215,6 +217,50 @@ export default function CourseOfferingEditor(props) {
           ))}
         </select>
       </label>
+      <label>
+        CS Topic
+        <HelpTip>
+          <p>
+            Select all related CS topics. Shift-click or cmd-click to select
+            multiple.
+          </p>
+        </HelpTip>
+        <select
+          multiple
+          value={courseOffering.cs_topic?.split(',')}
+          style={styles.dropdown}
+          onChange={e => handleMultipleSelected(e, 'cs_topic')}
+        >
+          <option value="">(None)</option>
+          {Object.values(CourseOfferingCsTopics).map(topic => (
+            <option key={topic} value={topic}>
+              {topic}
+            </option>
+          ))}
+        </select>
+      </label>
+      <label>
+        School Subject
+        <HelpTip>
+          <p>
+            Select all related school subjects. Shift-click or cmd-click to
+            select multiple.
+          </p>
+        </HelpTip>
+        <select
+          multiple
+          value={courseOffering.school_subject?.split(',')}
+          style={styles.dropdown}
+          onChange={e => handleMultipleSelected(e, 'school_subject')}
+        >
+          <option value="">(None)</option>
+          {Object.values(CourseOfferingSchoolSubjects).map(subject => (
+            <option key={subject} value={subject}>
+              {subject}
```

Comment:
this line would become something like `{schoolSubjectDisplayName(subject)}`, which would use a map to find the translated name